helios-integration-tests/src/test/resources/logback-test.xml

```diff
@@ -9,7 +9,6 @@
   </appender>
 
   <logger name="com.spotify.helios.client" level="info"/>
-  <logger name="com.spotify.hermes" level="info"/>
```

Comment:
If you're cleaning up, maybe remove `<logger name="com.spotify.helios.client" level="info"/>` as well then?